packages/common/src/utils/url-generator.ts

```diff
@@ -32,6 +32,7 @@ export const parcelSandboxUrl = () => `/s/vanilla`;
 export const newReactTypeScriptSandboxUrl = () => `/s/react-ts`;
 export const newDojoSandboxUrl = () =>
   `/s/github/dojo/dojo-codesandbox-template`;
+export const newMarkoSandboxUrl = () => `s/marko`;
```

Comment:
I don't think you need this part of the code, since it's used nowhere. 
```suggestion
```

Reply:
deleted.